src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -221,12 +238,10 @@ public void setTags(Set<Tag> tags) {
         }
 
         /**
-         * Returns an unmodifiable tag set, which throws {@code UnsupportedOperationException}
-         * if modification is attempted.
          * Returns {@code Optional#empty()} if {@code tags} is null.
          */
         public Optional<Set<Tag>> getTags() {
-            return (tags != null) ? Optional.of(Collections.unmodifiableSet(tags)) : Optional.empty();
+            return (tags != null) ? Optional.of(tags) : Optional.empty();
```

Comment:
Would this change mean that tags are now modifiable, and if so, would this go against the immutability of the other variables?

Reply:
Yes, tag is changed to mutable in order to support the removal of an existing net worth tag